resources/views/pages/home.blade.php

```diff
@@ -1,8 +1,188 @@
-        <h1 class="heading-h1">Waar en wanneer het jou uitkomt</h1>
-        <p class="text-eph-grey">Behandeld worden door professionele fysiotherapeuten door heel Nederland.</p>
-        <x-links.button class="btn-primary rounded-full mt-4" href="#">Hoe werkt het?</x-link-button>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+
+
+    <!--Hero section1-->
+    <section class="container m-auto px-6">
+        <div class="lg:flex justify-between items-center">
+            <div class="lg:w-6/12 lg:p-0 p-7">
+                <small class="text-eph-purple font-bold">Online fysiotherapie</small>
+                <h1 class="heading-h1">Waar en wanneer het jou uitkomt</h1>
+                <p class="text-eph-grey mt-6">Behandeld worden door professionele fysiotherapeuten door heel Nederland.</p>
+                <div class="py-5">
+                    <x-links.button class="btn-primary rounded-full mt-4 font-normal" href="#">Hoe werkt het?</x-link-button>
+                </div>
+            </div>
+              <div class="lg:w-5/12">
+                <img class="rounded" src="https://images.unsplash.com/photo-1517245386807-bb43f82c33c4?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80">
+              </div>
+        </div>
+    </section>
+
+
+
+    <!--Tags section2-->
+    <section class="container m-auto mt-36 mb-36 px-6">
+        <div class="flex items-center justify-center">
+            <div class="text-center">
+                <h1 class="heading-h5 py-5 mt-6">Online fysiotherapie is o.a. geschikt voor:</h1>
+                <div class="max-w-2xl mt-6">
+                    <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Fysiotherapeutisch advies</span>
+                    <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Lage rugklachten</span>
+                    <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Nekklachten</span>
+                    <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Gewrichtsklachten</span>
+                    <div class="pt-7">
+                        <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Sportblessures</span>
+                        <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Revalideren na een operatie</span>
+                        <span class="m-1 rounded-md py-3 px-2 font-bold text-sm cursor-pointer" style="background-color:#F6F1FE; color: #9F62F6;">Herstellen na COVID-19</span>
+                    </div>
```

Comment:
Hadden we deze kleuren nog niet ingesteld? Als die kleuren er nog niet zijn mag je wat mij betreft ook wel een `<style>`-tag erboven zetten met wat custom classes. Dat maakt de code wat leesbaarder.